velox/dwio/dwrf/test/CacheInputTest.cpp

```diff
@@ -641,27 +635,26 @@ TEST_F(CacheTest, ssdThreads) {
 class FileWithReadAhead {
  public:
   static constexpr int32_t kFileSize = 21 << 20;
-  static constexpr int64_t kLoadQuantum = 4 << 20;
```

Comment:
Why change 4 -> 6?

Reply:
Unit tests were failing with out of memory in the pool errors, needed to increase the size.